pkg/executables/cmk.go

```diff
@@ -26,17 +27,26 @@ type Cmk struct {
 	writer     filewriter.FileWriter
 	executable Executable
 	config     CmkExecConfig
+	zones []cmkZone
+	domain cmkDomain
+	account cmkAccount
```

Comment:
Why are we storing these in the cmk struct?

Reply:
removed/refactored after merging with latest main commits